src/components/singleArticleDetails.jsx

```diff
@@ -17,16 +17,17 @@ const SingleArticleDetails = () => {
   const [commentsHidden, setCommentsHidden] = useState(true);
   const [article, setArticle] = useState([]);
   const [voteCount, setVoteCount] = useState(article.votes);
-  const [commentCount, setCommentCount] = useState('');
+  const [commentCount, setCommentCount] = useState("");
   const [isCommentListError, setIsCommentListError] = useState(false);
```

Comment:
I think you should have a rethink about where some of these things are living. Is it necessary to have to comments in the singleArticlePage if you're just passing them down into the comments component? It would also make sense to separate the vote logic out into its own component.